Tests/XCUITests/ExperimentIntegrationTests.swift

```diff
@@ -0,0 +1,55 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import XCTest
+
+final class ExperimentIntegrationTests: BaseTestCase {
+    override func setUpApp() {
+        app.activate()
+        let closeButton = app.buttons["bottomSheet close"]
+        if closeButton.exists {
+            closeButton.tap()
+        }
+        super.setUpScreenGraph()
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testVerifyExperimentEnrolled() throws {
+        let toolbarClose = app.buttons.matching(NSPredicate(format: "label CONTAINS 'Close'"))
+
+        wait(forElement: toolbarClose.element, timeout: 15)
+        toolbarClose.element(boundBy: 0).tap()
+        navigator.goto(SettingsScreen)
+
+        // enable experiments secret menu
+        let element = app.tables.cells.containing(
+            NSPredicate(format: "identifier CONTAINS 'FxVersion'")
+        )
+        for _ in 0...5 {
+            element.element.tap()
+        }
+
+        // open experiments menu
+        let experiments = app.tables.cells.containing(NSPredicate(format: "label CONTAINS 'Experiments'"))
+        experiments.element.tap()
+
+        // match json experiment name
+        let experiment = app.tables.cells.containing(NSPredicate(format: "label CONTAINS 'Viewpoint'"))
+        XCTAssertNotNil(experiment)
+    }
+
+    func testMessageNavigatesCorrectly() throws {
```

Comment:
Question: Even though there are two tests, there was only 1 Passed test result at the end, is that expected? maybe one depend on the other... but I would have expected 2 Passed as there are two tests..

Reply:
Well for pytest, it counts as 1